frontends/ol-components/src/components/Dialog/Dialog.tsx

```diff
@@ -24,7 +26,7 @@ const Content = styled.div`
   margin: 28px 28px 40px;
 `
 
-const Actions = styled(DialogActions)`
+export const Actions = styled(DialogActions)`
```

Comment:
Upon rebasing this PR, I get the following error when hitting the dashboard:

```
The requested module './components/Dialog/Dialog' contains conflicting star exports for the name 'Actions' with the previous requested module './components/Card/ListCardCondensed'
```

It looks like both of these components export a style named `Actions`. I'm not sure why these are even exported, as they aren't imported anywhere as far as I can tell. We should likely just not export them.